omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -173,6 +173,7 @@
 elasticsearch['initial_cluster_join_timeout'] = 90
 elasticsearch['shard_count'] = 5
 elasticsearch['replica_count'] = 1
+elasticsearch['es_version'] = '7.9.1'
```

Comment:
Is this required?

Reply:
yes, For identifing the ES version before installation used in elasticsearch.yml.erb file.

Reply:
Here is what I was thinking about: https://github.com/chef/chef-server/blob/master/omnibus/files/private-chef-cookbooks/private-chef/libraries/helper.rb#L58 detects the version of elasticsearch (after it is running) and then created the index accordingly. But in case of es7, we seem to require this information inorder to start es.

The auto-detection is helpful to make sure anyone running external elasticsearch does not run an unsupported version of elasticsearch. Whether is should be in that helper library or a preflight check is perhaps a question for a different time.

But since we are already doing that - I think of this parameter as redundant. Would es 2,5 and 6 continue to work if the new settings for 7 were made default or does that block from them starting up?

Reply:
We should add this to the docs.

Reply:
Updated the docs